components/UserTestimonies/UserTestimonies.js

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { Table, Container, Button } from "react-bootstrap"
+import ExpandTestimony from "../ExpandTestimony/ExpandTestimony"
+import EditTestimony from "../EditTestimony/EditTestimony"
+import DeleteTestimony from "../DeleteTestimony/DeleteTestimony"
+import { useAuth } from "../../components/auth"
+import { useBill, usePublishedTestimonyListing } from "../db"
+import { useRouter } from "next/router"
+
+const TestimonyRow = ({ testimony }) => {
+  const { result: bill } = useBill(testimony.billId)
+  const router = useRouter()
+  const { user, authenticated } = useAuth()
+  const userUid = user ? user.uid : null
+  const userIsAuthor = user?.uid == testimony?.authorUid
+
+  if (!bill) {
+    return null
+  } else {
+    return (
+      <tr>
+        <td>{testimony.position}</td>
+        <td>
+          <Button
```

Comment:
update to match testimoniestable (https://github.com/codeforboston/advocacy-maps/pull/390)

Reply:
done